labelbox/schema/slice.py

```diff
@@ -34,12 +35,25 @@ class CatalogSlice(Slice):
     Represents a Slice used for filtering data rows in Catalog.
     """
 
+    @dataclass
+    class DataRowIdAndGlobalKey:
+        id: UniqueId
+        global_key: Optional[GlobalKey]
+
+        def __init__(self, id: str, global_key: Optional[str]):
+            self.id = UniqueId(id)
+            self.global_key = GlobalKey(global_key) if global_key else None
+
+    warnings.warn(
```

Comment:
Should this warning be moved inside of the `get_data_row_ids` method instead?

Reply:
thx made the change